templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -157,6 +157,11 @@ export class {{ service.name }}Client {
     const fallback = opts?.fallback ?? (typeof window !== 'undefined' && typeof window?.fetch === 'function');
     opts = Object.assign({servicePath, port, clientConfig, fallback}, opts);
     {%- endif %}
+    {%- if api.restNumericEnums %}
```

Comment:
Is this inside something like `if rest` so it only gets applied when we generate REST transport?

Reply:
As discussed - we generate almost the same code for REST and gRPC client (the only difference being the default transport), so we want to pass this option down to gax even if the client is generated as a gRPC one, because the customer might choose to use it as a REST client by passing `{fallback: 'rest'}` to the constructor.